homeassistant/components/flexit_bacnet/README.md

```diff
@@ -0,0 +1,39 @@
+# Flexit Nordic (BACnet) integration
```

Comment:
I don't think adding this file makes sense

Reply:
Where is a good place to add documentation for the integration internals then?

Reply:
Mmmm... `prometheus`, `tplink` and `zwave_js` also have a readme.
I suggest that you move it to a separate PR, indicating why you feel it adds value.

Either way, it should not be part of a bugfix PR

Reply:
Exactly. I asked in the #developers channel in discord and got the answer that some have added READMEs.

I have removed the commit and will create a separate PR for that.